arcus_orientation.md

```diff
@@ -841,32 +849,32 @@ It is a violation of the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-us
 </div>
 
 Many researchers use GitHub to archive and share code for their analyses, often as part of the process of publishing a scientific paper.
-You can always use repositories on CHOP's Enterprise GitHub from your Arcus Lab, whether or not you have internet disabled, but CHOP's Enterprise GitHub is insufficient for sharing code as part of a publication because it's visible only to people with CHOP credentials. 
+You can use repositories on CHOP's Enterprise GitHub from your Arcus Lab in Internet Disabled mode, but CHOP's Enterprise GitHub is insufficient for sharing code as part of a publication because it's visible only to people with CHOP credentials. 
 
-If you plan to use a repository on [www.github.com](https://github.com/)(public GitHub) to share the R code from your Arcus Lab, you will need to enable internet when it's time to push commits to the repository. 
-Remember that you do not need an internet connection to do most actions using Git, such as creating new commits or merging branches, so you can wait until it's time to push to [enable internet](#lab-status-pane), and then disable internet again afterward. 
+If you plan to use a repository on [www.github.com](https://github.com/) (public GitHub) to share the R code from your Arcus Lab, you will need to enable internet when it's time to push commits to the repository. 
+Remember that you do not need an internet connection to do most actions using Git, such as creating new commits or merging branches, so you can wait until it's time to push to [enable internet](#switching-modes), and then return to Internet Disabled mode again afterward. 
 
 
 ### Using Python with Internet Disabled
 
 Most functions in Python run locally -- once you install the package they're in, they're saved on your machine (or in this case, in your Arcus Lab) for you to use whenever you like, without requiring you to have an internet connection. 
-If you already have Python code written for your analysis, chances are all of that code will continue to work in exactly the same way whether or not you keep your lab in Internet Disabled Mode. 
+If you already have Python code written for your analysis, chances are all of that code will continue to work in exactly the same way whether or not you keep your lab in Internet Disabled mode. 
 
```

Comment:
```suggestion

<div class = "warning">
<b style="color: rgb(var(--color-highlight));">Warning!</b><br>

If your code stops working when you switch to Internet Disabled mode and you don't understand why, **stop** before enabling internet and [reach out for help](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/303) to better understand what the code is doing. 

</div>
```

We have this in the R section, but folks may not read that section if they only use python, so I think it makes sense to also include it here.